components/Navbar.js

```diff
@@ -1,109 +1,90 @@
-								{/* <li><Link href="/gm"><a>{gmData.date.quarter} GM</a></Link></li> */}
-								<MobileNavItem/>
-								<li><Link href="/events"><a>Events</a></Link></li>
-								<li><Link href="/sponsors"><a>Sponsors</a></Link></li>
-								<li><Link href="https://members.uclaacm.com"><a className="button button-transparent button-lg font-header">Member Login</a></Link>
-								</li>
-							</ul>
-						</div>
-					</div>
-				</div>
-			</div>
-		);
-	}
+  return(
+    <nav id="navbar">
+      <section id="nav-container">
+        <Link href="/">
+          <a id="nav-title" className="nav-section left" aria-label="acm home">
+            {/* TODO: resolve next/image issue */}
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <img src={'/images/acm_wordmark&logo.svg'} id="acm-logo" alt="ACM at UCLA Logo"></img>
+            {/* TODO: use next image without breaking deploy */}
+          </a>
+        </Link>
+        <section id="nav-items-container">
+          <div className={menuActive} id="hamburger" role="button" aria-label="navigation menu" aria-expanded={menuActive === 'active' ? true : false} tabIndex="0">
+            <div className="bar" />
+          </div>
+          <section className={menuActive} id="menu-modal">
+            <ul className={`nav-items ${menuActive}`} role="presentation">
+              <li><Link href="/about"><a className={linkActive === '/about' ? 'active' : ''}>About</a></Link></li>
+              <li><Link href="/committees"><a className={linkActive === '/commitees' ? 'active' : ''}>Committees</a></Link></li>
+              {/* <li><Link href="/gm"><a>{gmData.date.quarter} GM</a></Link></li> */}
+              <li id="committee-mobile-nav-item">
+                <ul className="committee-mobile-nav" role="presentation">
+                  <li><Link href={'/committees#studio'}><a className={linkActive === '/committees#studio' ? 'active' : ''}>Studio</a></Link></li>
+                  <li><Link href={'/committees#icpc'}><a className={linkActive === '/committees#icpc' ? 'active' : ''}>ICPC</a></Link></li>
+                  <li><Link href={'/committees#design'}><a className={linkActive === '/committees#design' ? 'active' : ''}>Design</a></Link></li>
+                  <li><Link href={'/committees#cyber'}><a className={linkActive === '/committees#cyber' ? 'active' : ''}>Cyber</a></Link></li>
+                  <li><Link href={'/committees#teachla'}><a className={linkActive === '/committees#teachla' ? 'active' : ''}>Teach LA</a></Link></li>
+                  <li><Link href={'/committees#w'}><a className={linkActive === '/committees#w' ? 'active' : ''}>W</a></Link></li>
+                  <li><Link href={'/committees#ai'}><a className={linkActive === '/committees#ai' ? 'active' : ''}>AI</a></Link></li>
+                  <li><Link href={'/committees#hack'}><a className={linkActive === '/committees#hack' ? 'active' : ''}>Hack</a></Link></li>
```

Comment:
As far as I can see, there are 3 ways we can change the active state of these links. I'm fine with any of them so I'm going to leave this up to you.

Option 1:
    We can use javascript to manipulate the dom, adding and removing the class.

Option 2:
    We can go with inline logic for a more static approach.

Option 3:
    We can have a wrapper component around the each link which will apply the logic based on props passed from each link.

I know we want to minimize dom manipulation so I would say that option 3 is our best way to avoid that while bringing the logic out of inline.

Reply:
Let's got with option 3, you make a good argument in its favor